src/network/networkd-link.c

```diff
@@ -1325,6 +1325,11 @@ static int link_set_bridge(Link *link) {
                 if (r < 0)
                         return log_link_error_errno(link, r, "Could not append IFLA_BRPORT_COST attribute: %m");
         }
+        if (link->network->priority != 0) {
```

Comment:
Actually using an invalid value by default makes sense. Thus I now set priority at 128, which means don't update the priority such that whichever priority the device is at, it would stick. This is similar to using 0 in the path_cost, as 0 is an invalid setting for the path_cost.